net/src/net.cpp

```diff
@@ -597,4 +707,4 @@ int luteopen_net(lua_State* L)
     lua_setreadonly(L, -1, 1);
 
     return 1;
-}
+}
```

Comment:
```suggestion
}

```

Do not strip the final line break.